components/Slider/src/MDCSlider.h

```diff
@@ -161,6 +161,28 @@ IB_DESIGNABLE
  */
 - (nullable UIColor *)backgroundTrackTickColorForState:(UIControlState)state;
 
+/**
+ Sets the thumb radius to use for the specified state.
+
+ In general, if a property is not specified for a state, the default is to use the @c
+ UIControlStateNormal value. If the @c UIControlStateNormal value is not set, then the property
+ defaults to a default value. Therefore, at a minimum, you should set the value for the normal
+ state.
+
+ @param thumbRadius The radius of the thumb.
+ @param state The state of the slider.
+ */
+- (void)setThumbRadius:(CGFloat)thumbRadius forState:(UIControlState)state;
```

Comment:
What is happening to `-thumbRadius`?  Does it just get assigned to the `.normal` state?

Reply:
Yes

Reply:
Please document the behavior in the header. Similar updates to the documentation was added for the other properties affected by the "stateful API" flag.

Reply:
I added documentation in the header for `thumbRadius` property.